tests/llm/utils/mock_dal.py

```diff
@@ -2,26 +2,34 @@
 import json
 import logging
 from pathlib import Path
-from typing import Dict, Optional
+from typing import Dict, Optional, List
 
 from pydantic import TypeAdapter
 
-from holmes.core.supabase_dal import SupabaseDal
+from holmes.core.supabase_dal import SupabaseDal, FindingType
 from holmes.core.tool_calling_llm import Instructions, ResourceInstructions
 from tests.llm.utils.test_case_utils import read_file
+from datetime import datetime, timezone
 
 
 class MockSupabaseDal(SupabaseDal):
     def __init__(
         self,
         test_case_folder: Path,
         issue_data: Optional[Dict],
+        issues_metadata: Optional[List[Dict]],
         resource_instructions: Optional[ResourceInstructions],
         generate_mocks: bool,
     ):
-        super().__init__(cluster="test")
+        try:
+            super().__init__(cluster="test")
+        except Exception:
```

Comment:
you might need to set this to mock initialize it or other places in the code might fail
```suggestion
        except Exception:
            self.enabled = True
            self.cluster = "test"
```